### PR TITLE
weekly highlights 2022-10-27

### PR DESCRIPTION
**Pull request by gianfranco-l**
collect contributions for weekly highlights 2022-10-27

### CODE REVIEW

File: src/content/changes/highlights/2022-10-27.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+date: 2022-10-27T10:00:00
+title: Highlights for the week ending October 27, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+
+## User interfaces
```
Comment (marians) [inline src/content/changes/highlights/2022-10-27.md:11]:
```suggestion
---

## Feedback wanted

- We are currently designing the kubctl-gs CLI command for deleting workload clusters. The specification is available in a public [issue](https://github.com/giantswarm/roadmap/issues/1559#issuecomment-1290288319). We'd love to see you add comments, ideas, and expectations.
- Our [survey](https://forms.gle/jpGjU2DRbKohoudc7) regarding cluster creation is still open. If your organization's perspective isn't included yet, you still have some time.

## User interfaces
```

File: src/content/changes/highlights/2022-10-27.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- We are currently designing the kubctl-gs CLI command for deleting workload clusters. The specification is available in a public [issue](https://github.com/giantswarm/roadmap/issues/1559#issuecomment-1290288319). We'd love to see you add comments, ideas, and expectations.
+- Our [survey](https://forms.gle/jpGjU2DRbKohoudc7) regarding cluster creation is still open. If your organization's perspective isn't included yet, you still have some time.
+
+## User interfaces
+
+
+## Apps
+
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-10-27.md:20]:
```suggestion
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.11.6](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0116---2022-10-19) updates policy-reporter version 2.10.1.
- [security-pack](https://github.com/giantswarm/security-pack) version [0.8.1](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#081---2022-10-25) includes the new versions of Kyverno and starboard-exporter described in this announcement.
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.6.2](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#062---2022-10-24) makes the exporter's `ServiceMonitor` relabelings user-configurable, enabling custom metric collection and aggregation use cases.

```

File: src/content/changes/highlights/2022-10-27.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+title: Highlights for the week ending October 27, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Feedback wanted
+
+- We are currently designing the kubectl-gs CLI command for deleting workload clusters. The specification is available in a public [issue](https://github.com/giantswarm/roadmap/issues/1559#issuecomment-1290288319). We'd love to see you add comments, ideas, and expectations.
```
Comment (marians) [inline src/content/changes/highlights/2022-10-27.md:12]:
```suggestion
- We are currently designing the kubectl-gs CLI command for deleting workload clusters. The specification is available in a public [issue](https://github.com/giantswarm/roadmap/issues/1559#issuecomment-1291771290). We'd love to see you add comments, ideas, and expectations.
```

**Comment by T-Kukawka** [comment]
nothing from Phoenix

File: src/content/changes/highlights/2022-10-27.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- We are currently designing the kubectl-gs CLI command for deleting workload clusters. The specification is available in a public [issue](https://github.com/giantswarm/roadmap/issues/1559#issuecomment-1291771290). We'd love to see you add comments, ideas, and expectations.
+- Our [survey](https://forms.gle/jpGjU2DRbKohoudc7) regarding cluster creation is still open. If your organization's perspective isn't included yet, you still have some time.
+
+## User interfaces
+
+
+## Apps
+- [kong-app](https://github.com/giantswarm/kong-app) version [v2.13.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#2130---2022-10-25) Update kong/kubernetes-ingress-controller to [2.7.0](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v2.7.0) and require at least Kubernetes 1.22.
+
```
Comment (mcharriere) [inline src/content/changes/highlights/2022-10-27.md:20]:
```suggestion
- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v0.8.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#080---2022-10-27) Update linkerd2-cni to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd2-cni`, dropping the `-app` suffix.
- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v0.8.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#080---2022-10-27) Update linkerd to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd-control-plane` following upstream's naming convention.
```

**Review by marians — APPROVED** [review]

**Review by ubergesundheit — APPROVED** [review]

**Pull request merged**